blocks/centered-content-switcher/centered-content-switcher.js

```diff
@@ -60,7 +60,9 @@ export default function decorate(block) {
     ul(
       { class: 'desktop-nav-list' },
       ...[...block.children].map((child, idx) => {
-        const thumbnailUrl = child.children[0].children[2].querySelector('img').src;
+        const thumbnailUrl = child.children[0].children[3].querySelector('img').src;
```

Comment:
Rather than selecting the url, can you grab the whole "picture" element? The idea is that it's optimised for each screen size.